src/pages/[platform]/build-a-backend/data/data-modeling/relationships/index.mdx

```diff
@@ -263,11 +245,11 @@ Amplify.API.mutate(ModelMutation.update(memberWithRemovedTeam),
 <InlineFilter filters={["swift"]}>
 
 ```swift
-let memberWithRemovedTeam = Member(id: existingMember.id, name: existingMember.name, team: nil)
-let memberRemoveResult = try await Amplify.API.mutate(request: .update(memberWithRemovedTeam))
-guard case .success = memberRemoveResult else {
-    print("API response: \(memberRemoveResult)")
-    return
+do {
+    existingMember.setTeam(nil)
+    let memberRemovedTeam = try await Amplify.API.mutate(request: .update(existingMember)).get()
+} catch {
+    print("Failed to remove team from member", error)
 }
```

Comment:
<img width="1006" alt="image" src="https://github.com/aws-amplify/docs/assets/1365977/2e352062-1cd3-4ee4-be49-ce80cd89abff">

There's no need to recreate the entire `Member` instance, developers can set the team to `nil` and pass it back in